about/index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
It's better to use lang="ru" here, because the current page is in Russian. This is for seo, accessibility and etc